pkg/client/condition.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//nolint:unused
```

Comment:
which function(s) is(are) unused?

Reply:
yes this would be my option: remove dead code and see if the build/tests fail.

Reply:
ok, found all their usage in the end, basically belonged to the tests only